test/E2ETest/Microsoft.Test.E2E.AspNet.OData/BulkOperation/BulkOperationTest.cs

```diff
@@ -978,13 +978,15 @@ public async Task PostEmployee_WithCreateFriends()
             StringContent stringContent = new StringContent(content: content, encoding: Encoding.UTF8, mediaType: "application/json");
             requestForPatch.Content = stringContent;
 
+            var expected = "Friends\":[{\"Id\":1001,\"Name\":\"Friend 1001\",\"Age\":31},{\"Id\":1002,\"Name\":\"Friend 1002\",\"Age\":32},{\"Id\":1003,\"Name\":\"Friend 1003\",\"Age\":33}]";
```

Comment:
is there a nested property here?

Reply:
Maybe you can add a test with one object and a nested navigation property within the object.

Reply:
In the request payload, `Friends` is a nested (navigation) property.
This test checked if the Friends nested property is in the response.